code401challenges/src/main/java/code401challenges/tree/BinaryTree.java

```diff
@@ -45,6 +47,24 @@
         return list;
     }
 
+//    printLevelOrder
+    public ArrayList<T> breadthFirstTraversal(TreeNode<T> root) {
```

Comment:
You don't need to take in the root as an argument. Our task is to print the whole tree, so we always want to start at `this.root`